andromeda-ui/components/ColoredButton.tsx

```diff
@@ -1,25 +1,15 @@
+import {buttonColorClassName} from "../components/ButtonColors";
```

Comment:
These are from the same directory, why not 
```suggestion
import {buttonColorClassName} from "./ButtonColors";
```
as in the backend directory files?

Reply:
I was just following the convention for how .tsx components are imported, but since this is a .ts file so this change sounds good to me.